src/components/Chart/resolveOptions.js

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable func-names */
+/* eslint-disable object-shorthand */
+/* eslint-disable no-underscore-dangle */
```

Comment:
we should avoid adding eslint exceptions, only in some situations, keep in mind that they encourage best practices